cmake/LinkTestFixtures.cmake

```diff
@@ -0,0 +1,14 @@
+function(LinkTestFixtures _target)
+    set(TESTFIXTURES_DIR "${CMAKE_SOURCE_DIR}/tests/TestFixtures")
+
+    file(READ "${TESTFIXTURES_DIR}/exported-symbols.txt" TESTFIXTURES_EXPORTED_SYMBOLS)
```

Comment:
I did a little poking around, and it appears `file(STRINGS "${TESTFIXTURES_DIR}/exported-symbols.txt" TESTFIXTURES_EXPORTED_SYMBOLS)` can read the contents of a the exported symbols list file and split its lines into a list. Can we use it instead?